command/operator_raft_autopilot_state.go

```diff
@@ -69,21 +82,20 @@ func (c *OperatorRaftAutopilotStateCommand) Run(args []string) int {
 		return 1
 	}
 
-	args = f.Args()
-	switch len(args) {
-	case 0:
-	default:
-		c.UI.Error(fmt.Sprintf("Incorrect arguments (expected 0, got %d)", len(args)))
-		return 1
-	}
-
```

Comment:
Why remove this?

Reply:
You are correct, this should be here. Adding it back